front-end/front-end.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove changes from the front-end module

Reply:
IDEA file should be in the gitignore.
Remove from git ingex please.